### PR TITLE
Bump p/p version

### PR DESCRIPTION
**Pull request by guineveresaenger**
Followup to https://github.com/pulumi/registry/issues/4202 to propagate docsgen fixes in pu/pu.

### CODE REVIEW

**Comment by sean1588** [comment]
thanks @guineveresaenger!! can we hold off on bumping this now though? I am waiting on getting some of the new constructor syntax formatting changes into pu/pu before we update the version, since right now it is in a state where we want to make some more adjustments before we ship it live.

**Review by iwahbe — APPROVED** [review]
@sean1588 I defer to you when we should bump the version. My 2 cents is that we can take the fix now and then bump again later.

**Comment by sean1588** [comment]
going to close this assuming we will get this bumped to the latest in the next day. If not I'll reopen this and push this out there, because at there are many valuable fixes wrapped up in this release and may not be worth waiting.

**Pull request closed without merging**
